src/docs/environment/index.mdx

```diff
@@ -89,23 +89,55 @@ computer is actually compiling Python!
 make setup-pyenv
 ```
 
+fish users will need to manually set some of the environment variables. This only needs to be done
+once.
+
+```shell {tabTitle: Fish}
+set -Ux PYENV_ROOT $HOME/.pyenv
+# fish >=3.2.0
+fish_add_path $PYENV_ROOT/bin
+# fish <3.2.0
+set -U fish_user_paths $PYENV_ROOT/bin $fish_user_paths
+```
+
 Once that's done, your shell needs to be reloaded. You can either reload it in-place, or close your terminal and start it
 again and cd into sentry. To reload it, run:
 
-```shell
+```shell {tabTitle: Bash}
+exec "$SHELL"
+```
+
+```shell {tabTitle: Zsh}
 exec "$SHELL"
 ```
 
+```shell {tabTitle: Fish}
+exec $SHELL
+```
+
 After this, if you type `which python`, you should see something like `$HOME/.pyenv/shims/python`
-rather than `/usr/bin/python` this is because the following has been added to your start up script:
+rather than `/usr/bin/python`. This is because the following has been added to your start up script:
 
-```shell
+
+```shell {filename: ~/.bash_profile} {tabTitle: Bash}
+eval "$(pyenv init --path)"
```

Comment:
I read the bash instructions and I decided to leave it to the user to follow the instructions (besides most engineers don't use bash).
```suggestion
Visit https://github.com/pyenv/pyenv#installation for instructions on how to set up Bash.
```